Makefile

```
@@ -16,11 +16,14 @@ NAME       := golang-builder
 VARIANTS   ?= base main
 VERSION    ?= 1.24
 
-all: build
+all: build test-unit
 
 build:
 	@./build.sh "$(VERSION)" "$(VARIANTS)"
 
+test-unit:
```

I prefer putting the `.PHONY` local.

```suggestion
.PHONY: test-unit
test-unit:
```